site/viamguides/src/install-esp32/install-esp32.md

```diff
@@ -0,0 +1,116 @@
+author: Joyce
+id: install-esp32
+summary: Instructions to install viam-micro-server on ESP32
+categories: Getting-Started, Developer
+environments: web
+status: Published
+feedback link: https://github.com/viam-labs/viamcodelabs/issues
+tags: Getting Started, Developer, Data
+
+# Install viam-micro-server on ESP32
+
+<!-- ------------------------ -->
+
+## Overview
+
+Duration: 1
+
+Learn how to install the `viam-micro-server` on an ESP32 board. It is a lightweight version of `viam-server` which can run on resource-limited embedded systems that cannot run the fully-featured `viam-server`. The `viam-micro-server` is built from the [micro-RDK](https://github.com/viamrobotics/micro-rdk/tree/main).
```

Comment:
Is there motivation behind this codelab? Why might a developer choose to use an ESP32 with the micro-server over a single board computer with the full viam-server?